ansible_image_validation/validate-vm-images.yaml

```diff
@@ -23,7 +23,7 @@
         err_folder: /tmp/err
         license_type: "{{ 'byos' if 'byos' in inventory_hostname else 'payg' }}"
         repo_type: "{{'base' if 'base' in offer_type else ('byol' if 'byol' in offer_type else 'eus')}}"
```

Comment:
I don't see any changes required in this file. Please revert them

Reply:
Done